Makefile

```diff
@@ -1,16 +1,61 @@
-.PHONY: docker 
+###############################################################################
+# Copyright 2019 VMWare.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+###############################################################################
 
-DOCKERS=docker_x86-64
+.PHONY: docker build clean
+
+DOCKERS=edgex-mongo
 .PHONY: $(DOCKERS)
 
+MICROSERVICES=cmd/edgex-mongo
+.PHONY: $(MICROSERVICES)
+
+GO=CGO_ENABLED=0 GO111MODULE=on go
+
+VERSION=$(shell cat ./VERSION)
+DOCKER_TAG=$(VERSION)-dev
```

Comment:
Will need to remove the "-dev" suffix so that blackbox testing will pick up the built images.